src/components/Map/FcsAccordion.tsx

```diff
@@ -0,0 +1,154 @@
+import { Spacer } from '@nextui-org/react';
+
+import FcsAccordionProps from '@/domain/props/FcsAccordionProps';
+import { FcsAccordionOperations } from '@/operations/map/FcsAccordionOperations';
+import { cardsWrapperClass } from '@/utils/primitives';
+
+import CustomAccordion from '../Accordions/Accordion';
+import CustomCard from '../Cards/Card';
+import { LineChart } from '../Charts/LineChart';
+
+export default function FcsAccordion({ countryData, loading, countryIso3Data }: FcsAccordionProps) {
+  const deltaOneMonth = countryData?.fcsMinus1 ? countryData.fcs - countryData.fcsMinus1 : undefined;
+  const deltaThreeMonth = countryData?.fcsMinus3 ? countryData.fcs - countryData.fcsMinus3 : undefined;
```

Comment:
useMemo here

Reply:
can we avoid using useMemo for simple number subtraction?